Source/Core/DolphinQt/CMakeLists.txt

```diff
@@ -396,6 +396,8 @@ if (MSVC)
   target_compile_options(dolphin-emu PRIVATE "${qtGuiPriv}")
   target_compile_options(dolphin-emu PRIVATE "${qtWidgets}")
 
+  target_compile_definitions(dolphin-emu PRIVATE _SILENCE_CXX23_ALIGNED_STORAGE_DEPRECATION_WARNING)
```

Comment:
Maybe add a comment like you did for the vcxproj:

`# Qt 6.3.0 headers use std::aligned_storage instead of alignas`